src/Authorization/PreventGuestWrapper.tsx

```diff
@@ -0,0 +1,73 @@
+import { ClipboardPen } from 'lucide-react';
+
+import {
+  Alert,
+  Box,
+  Container,
+  Button as MuiButton,
+  Stack,
+  Typography,
+} from '@mui/material';
+
+import { ReactNode } from 'react';
+
+import { AccountType, CurrentAccount } from '@graasp/sdk';
+
+type Props = {
+  buttonText?: string;
+  children?: ReactNode;
+  currentAccount?: CurrentAccount | null;
+  /**
+   * Component to display on error.
+   * Overrides errorText
+   */
+  error?: JSX.Element;
+  errorText?: string;
+  id?: string;
+  onButtonClick?: () => void;
+  startIcon?: JSX.Element;
+  text?: string | JSX.Element;
+};
+
+const PreventGuestWrapper = ({
+  buttonText = 'Log out and Create a Graasp account',
```

Comment:
I'm wondering if it's necessary to have defaults and to not define as required parameters. In any case, we'll have to redefine their to use the correct language right? Having their as optional add a risk to forgetting to define their with the user's language.